packages/jbrowse-plugin-apollo/src/ChangeManager.ts

```diff
@@ -22,22 +24,41 @@ export class ChangeManager {
   recentChanges: Change[] = []
 
   async submit(change: Change, opts: SubmitOpts = {}) {
-    const { addToRecents = true, submitToBackend = true } = opts
+    const {
+      addToRecents = true,
+      submitToBackend = true,
+      updateJobsManager = false,
+    } = opts
     // pre-validate
     const session = getSession(this.dataStore)
+    const { jobsManager } = session
```

Comment:
Getting the session vs Apollo session right can be tricky, but I think this should give you the right typings:

```suggestion
    const { notify } = getSession(this.dataStore)
    const { jobsManager } = getSession(
      this.dataStore,
    ) as unknown as ApolloSessionModel
```